bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -214,7 +214,11 @@ public static void leaveBreadcrumb(@NonNull String message,
      * Remove metadata from subsequent exception reports
      */
     public static void clearMetadata(@NonNull String section, @Nullable String key) {
-        getClient().clearMetadata(section, key);
+        if (key == null) {
+            getClient().clearMetadata(section);
+        } else {
+            getClient().clearMetadata(section, key);
+        }
```

Comment:
We seem to have lost something in the expressiveness of the API if we have to do this rather than rely on the method overrides, but I guess that's because we don't want an extra method in the native interface?

Reply:
We can add another method to the native interface if needed but I believe this would require updating downstream code in the NDK + Unity - supporting this with the existing method signature seemed the simplest option for now, and we could always alter the implementation at a later date.